app/src/routes/homeRoutes.tsx

```diff
@@ -0,0 +1,14 @@
+import RootComponent from "components/redirects/RootComponent";
+import PATHS from "config/constants/sub/paths";
+import { RouteObject } from "react-router-dom";
+
+export const homeRoutes: RouteObject[] = [
+  {
+    path: PATHS.ROOT,
+    element: <RootComponent />,
```

Comment:
I don't think we need both RootComponent and `useRedirectToLastFeature`. Only `useRedirectToLastFeature` will suffice. Can we remove the RootComponent and move its logic to `useRedirectToLastFeature` only